recipes/trame-leaflet/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "trame-leaflet" %}
+{% set version = "1.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/trame-leaflet-{{ version }}.tar.gz
+  sha256: 74d8b1ae0cce6195c17583c62ccdbeaae3b959f05b913de9e1bab2b7c1bd4262
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - trame-client
+
+test:
+  imports:
+    - trame
+    - trame_leaflet
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/Kitware/trame-leaflet
+  summary: Leaflet widget for trame
+  license: MIT
+  license_file:
+    - LICENSE
+    - trame_leaflet/LICENSE
```

Comment:
This is a symlink to the license file above. No need to add it here, please remove.

Reply:
Sorry about that